web/src/user_groups.ts

```diff
@@ -202,6 +202,23 @@ export function get_recursive_subgroups(target_user_group: UserGroup): Set<numbe
     return subgroup_ids;
 }
 
+export function get_recursive_group_members(target_user_group: UserGroup): Set<number> {
+    const members = new Set(target_user_group.members);
+    const subgroup_ids = get_recursive_subgroups(target_user_group);
+
+    if (subgroup_ids === undefined) {
+        return members;
+    }
```

Comment:
I'm returning the direct members of our `target_user_group` if we have an error while trying to find subgroups. I'm not sure in which circumstances we would end up with undefined in the real world scenario, so I've just decided to return the direct members for now. Let me know if this needs to change